packs/powerpoint/config.schema.yaml

```diff
@@ -0,0 +1,6 @@
+---
+  template:
+    description: "Path to PPTX template file"
+    type: "string"
+    secret: false
```

Comment:
False is a default value so if you prefer, you can leave `secret` attribute out all together for non-secret parameters.